pkgs/by-name/bo/bootspec-lix/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitea,
+  fetchpatch,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "bootspec-lix";
+  version = "1.0.0";
+
+  src = fetchFromGitea {
+    domain = "git.lix.systems";
+    owner = "lix-community";
+    repo = "bootspec";
+    rev = "v${version}";
+    hash = "sha256-5IGSMHeL0eKfl7teDejAckYQjc8aeLwfwIQSzQ8YaAg=";
+  };
+
+  patches = [
+    # https://github.com/DeterminateSystems/bootspec/pull/127
+    # Fixes the synthesize tool for aarch64-linux
+    (fetchpatch {
+      name = "aarch64-support.patch";
+      url = "https://github.com/DeterminateSystems/bootspec/commit/1d0e925f360f0199f13422fb7541225fd162fd4f.patch";
```

Comment:
Why not pull this patch into your repo? Now you're still depending on the repo you don't want to depend on?

Reply:
Immutable, but the patch will be (finally) merged.

Reply:
Wasn't the patch merged in 2023?

Reply:
It was but no release was done in the meantime.

Reply:
All the things you fetch with nix are immutable ;). Anyway, we started a release here: https://github.com/DeterminateSystems/bootspec/pull/180

Reply:
Tagged!